scripts/platforms/wellfound_apply.py

```
@@ -0,0 +1,93 @@
+import asyncio
+import json
+
+from playwright.async_api import async_playwright
+
+
+with open(
+    "applicant.json",
+    "r",
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)